app/controllers/pxe_controller.rb

```diff
@@ -251,7 +251,7 @@ def replace_right_cell(options = {})
         presenter.update(:form_buttons_div, r[
           :partial => "layouts/x_edit_buttons",
           :locals  => {
-            :record_id    => @edit[:rec_id],
+            :record_id    => @edit.try(:[], :rec_id),
```

Comment:
I am confused about this change, looks like `@edit` will not exist because the method that used to set that is being removed https://github.com/ManageIQ/manageiq-ui-classic/pull/7595/files#diff-b1bd37536670bed9dbfc7fbaf6ce7dec7e3219689d3a409fef6bde995ef02473L135
Do we even need to update form buttons when React form is being rendered?

Reply:
I'm pretty sure we still need to for the non-react pxe forms